pkg/gatewayserver/io/ws/lbslns/version.go

```diff
@@ -64,7 +64,7 @@ func (v Version) IsProduction() bool {
 }
 
 // GetRouterConfig gets router config for the particular version message.
-func (f *lbsLNS) GetRouterConfig(ctx context.Context, msg []byte, bandID string, fps map[string]*frequencyplans.FrequencyPlan, receivedAt time.Time) (context.Context, []byte, *ttnpb.GatewayStatus, error) {
+func (f *lbsLNS) GetRouterConfig(ctx context.Context, msg []byte, bandID string, fps map[string]*frequencyplans.FrequencyPlan, antennaGain int, receivedAt time.Time) (context.Context, []byte, *ttnpb.GatewayStatus, error) {
```

Comment:
This func signature is getting a bit nasty, eventually we should probably take a struct instead of all these arguments.

Reply:
Oh haha, just noticed that @adriansmares wrote exactly the same comment